### DISCUSSION TITLE
upload audio file to Bale

### DISCUSSION THREAD

**Discussion by ranch-hands** [kianahmadian/python-bale-bot#66] [Q&A] [answered]
hi 
please write a code snippet to play sounds

**Comment by kian-ahmadian** [comment]
سلام عزیز وقت بخیر
نمونه کدی برای ارسال صدا نیاز دارید؟

**Reply by ranch-hands** [reply]
yes, I want a code to reply to what user says with sounds please.
 also a code snippet to show menu to user and help him to pay in the bank portal .

thanks very much in advance

**Reply by kian-ahmadian** [reply, marked as answer]
برای ارسال درخواست پول میتونین از طریق کد زیر اقدام نمایید
https://github.com/python-bale-bot/python-bale-bot/blob/master/examples/invoice.py

همچنین برای ارسال فایل صوتی هم میتونید از تکه کد زیر استفاده کنید
```py
from bale import Bot, Message, InputFile

client = Bot("Your Token")

@client.event
async def on_ready():
    print(client.user, "is Ready!")
    
@client.event
async def on_message(message: Message):
    if message.content == "/audio":
        file = open('attachment.mp3', 'rb').read()
        audio = InputFile(file)
        return await message.reply_audio(audio=audio, caption="This is a simple audio")

client.run()
```

**Comment by spiderlike** [comment]
Bale guys made me proud once again as a smart expert in IT. Tell you what... I kinda feel more than happy to help you as a regular user, let alone an engineer who'll be effective in places like Bale. You people rock and shine. look forward to sharing knowledge with you based on mutually accepted terms.
